org.bndtools.launch/src/bndtools/launch/AbstractOSGiLaunchDelegate.java

```diff
@@ -294,6 +296,13 @@ public String[][] getClasspathAndModulepath(ILaunchConfiguration config) throws
 			classpathAndModulepath = new String[2][];
 			classpathAndModulepath[1] = new String[0];
 		}
+		if (classpathAndModulepath[1].length > 0) {
+			classpathAndModulepath[1] = Arrays.asList(classpathAndModulepath[1])
+				.stream()
+				.filter(mp -> !mp.endsWith("jrt-fs.jar"))
+				.collect(Collectors.toList())
+				.toArray(new String[0]);
```

Comment:
```suggestion
				.toArray(String[]::new);
```

No need to collect to a list and then an array. Just go straight to an array.